duckduckgo/Android
# Feature/david/design system app components sample

Task/Issue URL: https://app.asana.com/0/1174433894299346/1201421483575524

### Description
This PR updates the App Components sample to showcase the new views and templates

### Steps to test this PR

_Sample_
1. Build Internal Debug
2. Launch the app and open Settings
3. Open App Components
4. Check that all new components are rendered properly
